### PR TITLE
Update using-custom-database-tables.md

### PR DESCRIPTION
**Pull request by BobRay**
Corrects broken link to https://docs.modx.com/3.x/en/extending-modx/xpdo/custom-models/defining-a-schema/database-and-tables#defining-tables

### CODE REVIEW

File: en/extending-modx/tutorials/using-custom-database-tables.md [hunk 1]
```diff
@@ -158,7 +158,7 @@ The model tag and its attributes define a few different things about your Compon
 
 ---
 
-> For more details on MODX schemas, see [Defining a Schema | Database and Tables](custom-models/defining-a-schema/database-and-tables#defining-tables). This gives additional details on the Class, Table, and Extends attributes, as well as additional examples.
+> For more details on MODX schemas, see [Defining a Schema | Database and Tables]([custom-models/defining-a-schema/database-and-tables#defining-tables](https://docs.modx.com/3.x/en/extending-modx/xpdo/custom-models/defining-a-schema/database-and-tables#defining-tables)). This gives additional details on the Class, Table, and Extends attributes, as well as additional examples.
```
Comment (Ibochkarev) [inline en/extending-modx/tutorials/using-custom-database-tables.md]:
```suggestion
> For more details on MODX schemas, see [Defining a Schema | Database and Tables](extending-modx/xpdo/custom-models/defining-a-schema/database-and-tables#defining-tables)). This gives additional details on the Class, Table, and Extends attributes, as well as additional examples.
```

File: en/extending-modx/tutorials/using-custom-database-tables.md [hunk 2]
```diff
@@ -472,4 +472,4 @@ width="600" />
 *[Reverse Engineer xPDO Classes from Existing Database Table](extending-modx/xpdo/custom-models/generating-the-model/reverse-engineer "Reverse Engineer xPDO Classes from Existing Database Table")
 * [Creating Objects](https://docs.modx.com/3.x/en/extending-modx/xpdo/creating-objects "Creating Objects")
 * [Retrieving Objects](https://docs.modx.com/3.x/en/extending-modx/xpdo/retrieving-objects "Retrieving Objects")
-* [Add Many | Adding multiple child records with one "save"](https://docs.modx.com/3.x/en/extending-modx/xpdo/class-reference/xpdoobject/related-object-accessors/addmany "Adding multiple child records with one save")
\ No newline at end of file
+* [Add Many | Adding multiple child records with one "save"](https://docs.modx.com/3.x/en/extending-modx/xpdo/class-reference/xpdoobject/related-object-accessors/addmany "Adding multiple child records with one save")
```
Comment (Ibochkarev) [inline en/extending-modx/tutorials/using-custom-database-tables.md]:
```suggestion
* [Add Many | Adding multiple child records with one "save"](extending-modx/xpdo/class-reference/xpdoobject/related-object-accessors/addmany)
```

**Review by Ibochkarev — CHANGES_REQUESTED** [review]

File: en/extending-modx/tutorials/using-custom-database-tables.md [hunk 3]
```diff
@@ -472,4 +472,4 @@ width="600" />
 *[Reverse Engineer xPDO Classes from Existing Database Table](extending-modx/xpdo/custom-models/generating-the-model/reverse-engineer "Reverse Engineer xPDO Classes from Existing Database Table")
 * [Creating Objects](https://docs.modx.com/3.x/en/extending-modx/xpdo/creating-objects "Creating Objects")
 * [Retrieving Objects](https://docs.modx.com/3.x/en/extending-modx/xpdo/retrieving-objects "Retrieving Objects")
```
Comment (Ibochkarev) [inline en/extending-modx/tutorials/using-custom-database-tables.md]:
```suggestion
* [Retrieving Objects](extending-modx/xpdo/retrieving-objects)
```

File: en/extending-modx/tutorials/using-custom-database-tables.md [hunk 4]
```diff
@@ -472,4 +472,4 @@ width="600" />
 *[Reverse Engineer xPDO Classes from Existing Database Table](extending-modx/xpdo/custom-models/generating-the-model/reverse-engineer "Reverse Engineer xPDO Classes from Existing Database Table")
 * [Creating Objects](https://docs.modx.com/3.x/en/extending-modx/xpdo/creating-objects "Creating Objects")
```
Comment (Ibochkarev) [inline en/extending-modx/tutorials/using-custom-database-tables.md]:
```suggestion
* [Creating Objects](extending-modx/xpdo/creating-objects)
```

**Review by Ibochkarev — CHANGES_REQUESTED** [review]

**Comment by BobRay** [comment]
These all look good to me.

**Pull request merged**